test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -5,7 +5,7 @@
 import test.test_utils.ec2 as ec2_utils
 
 from test import test_utils
-from test.test_utils import CONTAINER_TESTS_PREFIX, get_framework_and_version_from_tag
+from test.test_utils import CONTAINER_TESTS_PREFIX, get_framework_and_version_from_tag, get_cuda_version_from_tag
```

Comment:
`get_cuda_version_from_tag` is this fixture used?

Reply:
You are right. I can remove this. Is this the only change needed to make it Approval-ready?

Reply:
Approved!